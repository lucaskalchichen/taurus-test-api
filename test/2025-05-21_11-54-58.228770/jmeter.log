2025-05-21 11:55:01,964 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 11:55:01,974 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 11:55:01,974 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 11:55:01,975 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770/jmeter-bzt.properties
2025-05-21 11:55:01,976 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770/system.properties
2025-05-21 11:55:01,979 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 11:55:01,979 INFO o.a.j.JMeter: Version 5.5
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 11:55:01,980 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 11:55:01,981 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 11:55:01,983 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 11:55:01,983 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 11:55:01,984 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 11:55:01,984 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:55:01,984 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 11:55:01,990 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 11:55:01,990 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770
2025-05-21 11:55:01,990 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770
2025-05-21 11:55:01,991 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770
2025-05-21 11:55:01,997 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 11:55:01,997 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770'
2025-05-21 11:55:02,076 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 11:55:02,087 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 11:55:02,088 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 11:55:02,090 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770/modified_requests.jmx
2025-05-21 11:55:02,106 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 11:55:02,107 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 11:55:02,114 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 11:55:02,130 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 11:55:02,131 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:55:02,131 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 11:55:02,136 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 11:55:02,137 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 11:55:02,265 INFO o.a.j.JMeter: Running test (1747839302265)
2025-05-21 11:55:02,274 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 11:55:02,274 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 11:55:02,274 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 11:55:02,274 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 11:55:02,280 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 11:55:02,292 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 11:55:02,292 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 11:55:02,293 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 11:55:02,294 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 11:55:02,295 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 11:55:02,295 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 11:55:02,295 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 11:55:02,295 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 11:55:02,348 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 11:55:02,417 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 11:55:02,490 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 11:55:02,561 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 11:55:02,631 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 11:55:02,700 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 11:55:02,772 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 11:55:02,843 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 11:55:02,914 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 11:55:02,985 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 11:55:03,056 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 11:55:03,126 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 11:55:03,198 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 11:55:03,268 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 11:55:03,340 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 11:55:03,410 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 11:55:03,477 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 11:55:03,477 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 11:55:03,483 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 11:55:03,553 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 11:55:03,623 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 11:55:03,695 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 11:55:03,766 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 11:55:03,837 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 11:55:03,865 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 11:55:03,865 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 11:55:03,908 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 11:55:03,978 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 11:55:04,050 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 11:55:04,121 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 11:55:04,192 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 11:55:04,263 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 11:55:04,308 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 11:55:04,308 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 11:55:04,334 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 11:55:04,404 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 11:55:04,476 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 11:55:04,547 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 11:55:04,617 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 11:55:04,689 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 11:55:04,760 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 11:55:04,830 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 11:55:04,830 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 11:55:04,830 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 11:55:04,902 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 11:55:04,973 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 11:55:05,043 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 11:55:05,114 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 11:55:05,186 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 11:55:05,370 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 11:55:05,370 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 11:55:05,723 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 11:55:05,724 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 11:55:06,207 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 11:55:06,207 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 11:55:06,545 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 11:55:06,545 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 11:55:06,990 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 11:55:06,991 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 11:55:07,510 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 11:55:07,510 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 11:55:08,041 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 11:55:08,041 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 11:55:08,405 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 11:55:08,406 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 11:55:08,851 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 11:55:08,852 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 11:55:09,222 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 11:55:09,222 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 11:55:09,663 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 11:55:09,663 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 11:55:10,173 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 11:55:10,173 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 11:55:10,672 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 11:55:10,672 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 11:55:11,023 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 11:55:11,023 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 11:55:11,451 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 11:55:11,451 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 11:55:11,927 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 11:55:11,927 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 11:55:12,284 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 11:55:12,284 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 11:55:12,688 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 11:55:12,688 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 11:55:13,155 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 11:55:13,155 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 11:55:13,612 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 11:55:13,612 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 11:55:14,099 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 11:55:14,100 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 11:55:14,556 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 11:55:14,556 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 11:55:15,027 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 11:55:15,027 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 11:55:15,481 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 11:55:15,482 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 11:55:15,936 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 11:55:15,936 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 11:55:16,392 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 11:55:16,392 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 11:55:16,863 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 11:55:16,863 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 11:55:17,332 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 11:55:17,332 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 11:55:17,786 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 11:55:17,786 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 11:55:18,244 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 11:55:18,244 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 11:55:18,766 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 11:55:18,766 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 11:55:19,215 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 11:55:19,215 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 11:55:19,670 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 11:55:19,670 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 11:55:20,121 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 11:55:20,121 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 11:55:20,593 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 11:55:20,594 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 11:55:21,050 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 11:55:21,051 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 11:55:21,541 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 11:55:21,541 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 11:55:21,946 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 11:55:21,946 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 11:55:21,946 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 11:55:21,948 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
