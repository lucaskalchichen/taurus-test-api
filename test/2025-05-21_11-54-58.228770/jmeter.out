Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_11-54-58.228770/modified_requests.jmx
Starting standalone test @ 2025 May 21 11:55:02 ART (1747839302126)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 11:55:21 ART (1747839321947)
... end of run
